docs/admin/admin-ui/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+- installation
+- license
+---
+
+# Overview
```

Comment:
Consider changing the title to `Admin UI` instead of `Overview`. It helps in search. This should be standard practice to use the relevant titles instead of `overview`

Reply:
Renamed title to `Gluu Flex Admin UI`. Also provide appropriate title name in other pages.